lineage/sub-lineage.xml

```diff
@@ -0,0 +1,95 @@
+<gmd:LI_Lineage xmlns:gml="http://www.opengis.net/gml/3.2"
+                xmlns:gts="http://www.isotc211.org/2005/gts"
+                xmlns:gmd="http://www.isotc211.org/2005/gmd"
+                xmlns:gmx="http://www.isotc211.org/2005/gmx"
+                xmlns:gco="http://www.isotc211.org/2005/gco"
+                xmlns:xlink="http://www.w3.org/1999/xlink">
+    <gmd:statement>
+        <gco:CharacterString>Lineage with processing step</gco:CharacterString>
+    </gmd:statement>
+    <gmd:processStep>
+        <gmd:LI_ProcessStep>
+            <gmd:description>
+                <gco:CharacterString>A description of an event or transformation in the life of a dataset- at minimum, the process used to maintain the dataset.</gco:CharacterString>
+            </gmd:description>
+            <gmd:rationale>
+                <gco:CharacterString>Requirement or purpose for the process step</gco:CharacterString>
+            </gmd:rationale>
+            <gmd:dateTime>
+                <gco:DateTime>2023-01-01T00:00:00+00:00</gco:DateTime>
+            </gmd:dateTime>
+            <gmd:processor>
+                <gmd:CI_ResponsibleParty>
+                    <gmd:organisationName>
+                        <gco:CharacterString>Name of the responsible organization</gco:CharacterString>
```

Comment:
I think I would probably strip out as much of the contact details as you can get way with but still create a valid record, to try and keep it as brief as possible. I'd also make the descriptions (eg Name of the responsible organisation, etc) be more targeted to the lineage process rather than generic